app/controllers/events_controller.rb

```diff
@@ -0,0 +1,29 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = current_user.created_events.build
+    @event.tittle = params[:event][:tittle]
+    @event.description = params[:event][:description]
+    @event.date = params[:event][:date]
+    @event.location = params[:event][:location]
+    if @event.save
+      flash[:message] = 'New event created'
+      redirect_to event_path(@event)
+    else
+      flash.now[:message] = 'Something went wrong'
+      render 'new'
+    end
+  end
+
+  def show
+    @event = Event.find_by(id: params[:id])
+    @event.attendees = User.all
+  end
+
+  def index
+    @created_events = Event.all
+  end
```

Comment:
[Required] Should have instance variables for upcoming and past events. [reference](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png)